runner.py

```diff
@@ -0,0 +1,68 @@
+from character import Character
+from wordle import playWordle
+from item import Item0
+from steps import stepcounter
+import steps 
+
+
+flashlight = Item0("Flashlight", "Light")
+
+Map = Item0("Map", "Direction")
+
+
+maincharacter = Character("", 100, [], "" )
+maincharacter.name = input("Tell me your name brave warrior. ")
+maincharacter.rizz = input("What level of Looksmax Mogger are you? ")
+
+
+npccharacter = Character("The Traveler", 100, [], "")
+
+walk = input("You are lost in a forest, where would you like to go? N/S/E/W? ")
+
+
+print("Welcome to the FOREST OF DOOM!!! Figure your way out of the forest before the skinwalker gets you. Work quick before you run out of steps. ")
+    
+
+
+
+if walk == "N":
+
+    flashlightopp = input("You come up to a farm and see a flashlight. Would you like to have it in your inventory. (y/n)")
+
+    if flashlightopp == "y":
+        stepcounter()
+        maincharacter.inventory.append(flashlight.name)
+        walk = input("You are still lost in a forest, where would you like to go? N/S/E/W? ")
+    else:
+        print("You are lost in the forest. Game Over.")
+
+    print(steps)
+    
+
+elif walk == "S":
+    mapopp = input(npccharacter.name + " Hello Fellow Traveler, I am the Traveler. I am here to help you find your way out of the forest. I have a map that will help you find your way out of the forest. Would you like to have it in your inventory? (y/n)")
+   
+    if mapopp == "y":
+        stepcounter()
+        maincharacter.inventory.append(Map.name)
```

Comment:
```suggestion
print("In order to earn the map - you must play a game of wordle!")
win = playWordle("QUEST")
if win == true:
      maincharacter.inventory.append(Map.name)
else:
      print("You were not able to get the map.")
```
Make sure you look at the suggestions I made in your wordle file too.